level.go

```diff
@@ -78,19 +81,20 @@ func AddModuleLevel(backend Backend) LeveledBackend {
 	var leveled LeveledBackend
 	var ok bool
 	if leveled, ok = backend.(LeveledBackend); !ok {
-		leveled = &moduleLeveled{
-			levels:  make(map[string]Level),
-			backend: backend,
-		}
+		modLeveled := &moduleLeveled{}
+		modLeveled.levels.Store(make(moduleLeveledMap))
+		modLeveled.backend.Store(backend)
```

Comment:
Don't think need to hold Store in sync.Value as it's never going to race as only written during creation